.github/workflows/test_simple.sh

```diff
@@ -3,4 +3,19 @@
 # fail on non-zero return code from a subprocess
 set -e
 
+echo $PATH
+# Test if grass executable is found
+which grass
+grass_exe=$(which grass)
```

Comment:
I always find `exe` in non-Windows/cross platform context confusing. I see it used mostly in tools with some strong relation to Windows world such as Mono. `executable` is a fine name (see e.g. `sys.executable` below).

Reply:
With .github/workflows/print_versions.sh this is not needed any more...